api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java

```diff
@@ -176,21 +176,21 @@ public void setUp() throws Exception {
     cdrVersion = new CdrVersion();
     cdrVersion.setBigqueryDataset(testWorkbenchConfig.bigquery.dataSetId);
     cdrVersion.setBigqueryProject(testWorkbenchConfig.bigquery.projectId);
-    cdrVersionDao.save(cdrVersion);
+    cdrVersion = cdrVersionDao.save(cdrVersion);
 
     workspace = new DbWorkspace();
     workspace.setCdrVersion(cdrVersion);
     workspace.setWorkspaceNamespace(NAMESPACE);
     workspace.setFirecloudName(NAME);
-    workspaceDao.save(workspace);
+    workspace = workspaceDao.save(workspace);
```

Comment:
nit: can we rename the variable to dbWorkspace?

Reply:
Yes, I had that in this PR but it drowned out the save-related changes (some of which are subtle).